src/backend/distributed/sql/citus--11.0-1--11.0-2.sql

```diff
@@ -1 +1 @@
--- bump version to 11.0-2
+#include "udfs/citus_is_coordinator/11.0-2.sql"
```

Comment:
The current version is '11.1-1', so this should be in citus--11.0-2--11.1-1.sql ?

Reply:
You are dropping from citus--11.1-1--11.0-2.sql

Reply:
We will backport this to release-11.0 which is currently at 11.0-1 (for the beta).